packages/polaris-viz/src/components/FunnelChart/components/Label.scss

```diff
@@ -1,14 +1,17 @@
 .Label {
   text-align: center;
   padding-top: 6px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 
-  span {
+  > span {
     padding: 0 4px;
     border-radius: '4px';
     border-radius: 4px;
   }
 
-  span:empty {
+  > span:empty {
     display: none;
   }
```

Comment:
Added more specificity with `>` because without it, it was also targeting the span element inside the Tooltip component